path/path.go

```diff
@@ -0,0 +1,176 @@
+package path
+
+import (
+	"bytes"
+	"database/sql/driver"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+
+	uuid "github.com/satori/go.uuid"
+)
+
+// LtreePath type helps in storing and retrieving ltree fields to and from databse.
+// It uses ltree knowledge while converting values. https://www.postgresql.org/docs/9.1/static/ltree.html
+// It is just slice of UUIDs which will be saved to database in following format
+// 4dd8f038_3fc4_48ab_ad4d_197ccc7b44a2.62ea5454_f8d5_4b35_8589_8d646d612250.c9b24b8f_8b33_4c22_82f0_8eb0a5b9837e
+// Above content will be read from database as a slice of UUIDs
+type LtreePath []uuid.UUID
+
+// Following constants are used while saving and displaying LtreePath type to and from database
+const (
+	SepInService  = "/"
+	SepInDatabase = "."
+)
+
+// IsEmpty checks count of items in LtreePath
+func (p LtreePath) IsEmpty() bool {
+	return len(p) == 0
+}
+
+// This returns last UUID of slice
+func (p LtreePath) This() uuid.UUID {
+	if len(p) > 0 {
+		return p[len(p)-1]
+	}
+	return uuid.UUID{}
```

Comment:
What I found is `uuid.Nil == uuid.UUID{}`
Yeah I can change it to `return uuid.Nil`. Thanks.